chrome_extension/export_messages.py

```diff
@@ -0,0 +1,64 @@
+"""Script to export messages from the database to an HTML file."""
```

Comment:
What do you think about moving this to `scripts/export_chrome_messages.py`?

Also, how about adding a line about why this is useful, and another one with example usage?